More ToT updates - primarily spill planning

tgross-usgs opened the pull request:
Changes include:

- update to accumulation process to avoid reach divergences that have bad VAA data and therefore bad travel time calculations
- updates to the spill planning unit checker to modify properties of reaches with bad data to "N/A" so they will also be accepted into the footer table
- updates to reach colors/styles. Connectors, canals and ditches are now colored according to their accumulated peak travel time values but are noticeably dashed. They were originally set to gray like waterbodies. Now, along with waterbodies, reach segments with bad VAA table data are set to gray. 
- fixed the overland trace dash (was showing as a solid line)
- updates to reach endnode style so they aren't so huge when zooming in
- cleanup of appcharts chart options
- updates the highlight feature function so that colors are no longer overridden and so that the highlighted feature is now purple since red was difficult to discern amongst red spill planning reaches

Feel free to test random locations with both the spill planning and the spill response method. If no issues arise and table values look normal we can move to test. Pete was getting NaN values at bad data locations and they transferred to every reach upstream of a bad reach so widespread bad data essentially. Notice below the velocity value of NaN. It couldn't be calculated due to bad data. Every reach after the bad reach was ruined with our accumulation process. This is the most important fix in this pull request. 
![image](https://user-images.githubusercontent.com/28792589/172740498-7d09ac54-a8b9-470d-95a9-d93b94092642.png)
If you want to test this area to compare for improvement the click-point is just south of Joliet, MT
![image](https://user-images.githubusercontent.com/28792589/172740553-891369b9-a29b-4224-b224-c706fa5677df.png)

tgross-usgs committed:
improves accumulation where bad data is present, improves colorization of bad data and canals, ditches, and connectors, resizes reach endpoints, cleans up appcharts graph options

tgross-usgs committed:
removes secrets from dist

harper-wavra reviewed (changes requested):
Seems to be working well! A couple things I noticed 
- I tried that point south of Joliet, MT (with spill planning) and when I opened the report the maximum probable table was empty. (This was happening in multiple places for spill planning, it seems like spill response was showing everything correct)
[maximum probable](https://user-images.githubusercontent.com/59063181/172854590-39a40b3b-c5a9-4942-bd56-bf2694ed10c0.png)
- If I zoom in past level 16 the base map goes away. This is happening on the beta and test site too. Seems like its only a problem with the National Geographic layer. Maybe that base map changed its max zoom level?

tgross-usgs committed:
cleans up tables in report to accomodate non-numeric values, sets maxzoom on map to 16 (the max offered by our default base layer)

tgross-usgs commented:
Excellent catch on the report tables. They are now cleaned up to accommodate non-numeric values. I also like that you pointed out the zoom issue with the National Geographic Layer. I updated the maxzoom of the map so users don't lose their basemap. Thanks for a speedy review!

tgross-usgs committed:
saves dist changes

tgross-usgs committed:
removes secrets

tgross-usgs committed:
Merge branch 'dev' into TAG-spillPlanning
